panel/command/serve.py

```diff
@@ -670,5 +670,9 @@ def invoke(self, args: argparse.Namespace):
         # Empty layout are valid and the Bokeh warning is silenced as usually
         # not relevant to Panel users.
         silence(EMPTY_LAYOUT, True)
+        # dask.distributed changes the logging level of Bokeh, we will overwrite it
```

Comment:
It'd be nice to link to one of the issues related to this or to this PR https://github.com/dask/distributed/pull/8634.

Reply:
Done in https://github.com/holoviz/panel/pull/7426/commits/97353bbab0dd34e1fe6d81c5ab6a524423a64de3